packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -71,7 +80,9 @@ export const _videoGalleryRemoteParticipantsMemo: (
             state,
             displayName,
             /* @conditional-compile-remove(raise-hand) */
-            participant.raisedHand
+            participant.raisedHand,
+            /* @conditional-compile-remove(reaction) */
+            remoteParticipantReaction
```

Comment:
(same as other comment on ensuring this param is memoized)

Reply:
Updated accordingly.